app/models/admin_set.rb

```diff
@@ -33,6 +32,7 @@ class AdminSet < Valkyrie::Resource
   attribute :title, Valkyrie::Types::Set
   attribute :description, Valkyrie::Types::Set
   attribute :creator, Valkyrie::Types::Set
+  attribute :thumbnail_id, Valkyrie::Types::SingleValuedString
 
```

Comment:
Is representative_id needed here as well?

Reply:
Not for an AdminSet.

Reply:
Ok, just wanted to make sure since it had it with the mixin.